js/dashboard.js

```diff
@@ -50,142 +42,139 @@ const dashboardScreen = (meetings) => {
+
+                              </tr>                                `;
+                    })}
                     
                 </tbody>
             </table>
-        </div>` : 
-        `<div style="text-align: center; margin-top: 10rem"><h5>No Meeting Room</h5></div>`
+        </div>`
+            : `<div style="text-align: center; margin-top: 10rem"><h5>No Meeting Room</h5></div>`
         }
-    </div>` 
-    dashboardScreen.innerHTML = dashboardScreenHtml;
-}
-
-// // Emty checker
-// const isEmpty = (value) => {
-//     return value === undefined || value === null || value === [];
-// }
-
-// // Add Room
-// let submitRoom = () => {
-//     let roomId = document.getElementById("roomId").value;
-//     let roomName = document.getElementById("roomName").value;
-//     if(!roomId && !roomName){
-//         alert("Please, Fill all Field");
-//     } else {
-//         let rooms = JSON.parse(localStorage.getItem('rooms'));
-    
-//         if(rooms){
-//             rooms.push({roomId, roomName, roomStatus:  "Available", roomSlots: [] });
-//             localStorage.setItem("rooms", JSON.stringify(rooms));
-//         } else{
-//             let rooms = [];
-//             rooms.push({roomId, roomName, roomStatus:  "Available", roomSlots: [] });
-//             localStorage.setItem("rooms", JSON.stringify(rooms));
-//         }
-//         document.getElementById("roomId").value = "";
-//         document.getElementById("roomName").value = "";
-//         alert("Room added Succesfully")
-//         location.reload()
-//     }
-// }
-
-// // Add Slot
-// const addSlotFunction = (editRoomId) => {
-//     localStorage.setItem("editRoomId", editRoomId);
-// }
-
+    </div>`;
+  dashboardScreen.innerHTML = dashboardScreenHtml;
+};
 
-// const addSlot = () => {
-//     let roomSlotDate = document.getElementById("roomSlotDate").value;
-//     let roomSlotStartTime = document.getElementById("roomSlotStartTime").value;
-//     let roomSlotEndTime = document.getElementById("roomSlotEndTime").value;
-//     let roomSlotId = document.getElementById("roomSlotId").value;
-//     let editRoomId = localStorage.getItem("editRoomId");
+// Add Meeting
+const submitMeeting = () => {
+  let meeting_name = document.getElementById("meetingTitle").value;
+  let fk_room_id = document.getElementById("roomType").value;
+  let meeting_date = document.getElementById("datepicker").value;
+  let start_time = document.getElementById("clockInTime").value;
+  let end_time = document.getElementById("clockOutTime").value;
+  let fk_emp_id = 1;
 
+  if (
+    !meeting_name &&
+    !fk_room_id &&
+    !meeting_date &&
+    !start_time &&
+    !end_time
+  ) {
+    alert("Please, Fill all Field");
+  } else {
+    console.log(meeting_date);
+    axios
+      .post("http://localhost:8080/meeting/add", {
+        meeting_name,
```

Comment:
create a separate variable and pass in API param

Reply:
done